src/xenia/kernel/util/xex2.cc

```diff
@@ -610,6 +610,7 @@ int xe_xex2_read_image_basic_compressed(const xe_xex2_header_t *header,
     return 1;
   }
   uint8_t *buffer = memory->TranslateVirtual(header->exe_address);
+  std::memset(buffer, 0, total_size); // Quickly zero the contents.
   uint8_t *d = buffer;
   std::memset(buffer, 0, uncompressed_size);
```

Comment:
Portal crashes without this - I'm guessing the BSS section isn't compressed at all.